src/controller/index.ts

```diff
@@ -13,32 +12,22 @@ import {
     FILE_STATUS_SUCCESS,
     FILE_STATUS_FAIL,
     PlotConfig,
-    Metrics,
 } from "../simularium/types.js";
 
-import { ClientSimulator } from "../simularium/ClientSimulator.js";
-import { IClientSimulatorImpl } from "../simularium/localSimulators/IClientSimulatorImpl.js";
-import { ISimulator } from "../simularium/ISimulator.js";
-import { LocalFileSimulator } from "../simularium/LocalFileSimulator.js";
+import {
+    getSimulatorClassFromParams,
```

Comment:
prob could just define getSimulatorClassFromParams in here, or make a new ts module called SimulatorFactory.ts